src/components/AppComponent.native.tsx

```diff
@@ -82,13 +85,21 @@ const AppComponent: React.FC = () => {
     height: height - 125,
   }
 
+  const theme = useTheme() as Theme
+
   return (
     <>
       <StatusBar
         // eslint-disable-next-line react/style-prop-object
         style='light'
       />
-      <SafeAreaView style={[flexOne, darkBackground]}>
+      <SafeAreaView
+        style={tw.style({
+          'bg-black': theme.mode === 'dark',
+          'bg-white': theme.mode === 'light',
```

Comment:
Why does this not use `themeSpecific`? It looks like it is doing the same kind of thing, but maybe I'm confused about where `themeSpecific` can be used.